### PR TITLE
Added Cuckoo Hashing Algorithm

### PR DESCRIPTION
**Pull request by shivhek25**
Solves issue #189

### CODE REVIEW

**Comment by aditya81070** [comment]
@shivhek25 squash commits.

**Comment by shivhek25** [comment]
@vikasgola  @sahilbansal17 Kindly review.

File: Algorithms/Hashing/cuckoo_hashing.cpp [hunk 1]
```diff
@@ -1 +1,88 @@
...
+// Email ID : shivamsinghal0610@gmail.com
 
+// Cuckoo hashing algorithm to improve worst case time for lookup in a table
+// based on multiple-choice hashing and relocation hashing
+
+#include <iostream>
+using namespace std;
+
+int indexTable[2];
+int n = 10;
```
Comment (vikasgola) [inline Algorithms/Hashing/cuckoo_hashing.cpp]: What this n stands for

Comment (shivhek25) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: Alright

Comment (shivhek25) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: @vikasgola I updated the file in my local repository but the change isn't reflecting here.

Comment (vikasgola) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: Commit your changes and push it to remote.

Comment (shivhek25) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: The PR says that it has been committed from an unknown repo.

Comment (vikasgola) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: Did the changes reflected to your forked repo?

Comment (shivhek25) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: Yes

Comment (shivhek25) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: screen shot 2018-07-22 at 2 26 07 pm

@vikasgola

**Comment by shivhek25** [comment]
@vikasgola Kindly suggest as to how should I proceed with this PR.

File: Algorithms/Hashing/cuckoo_hashing.cpp (hunk 1 above)
Comment (vikasgola) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: @shivhek25 make a new pull request.

Comment (vikasgola) [reply Algorithms/Hashing/cuckoo_hashing.cpp]: @shivhek25 and also take n input from user instead of making it 10.

**Pull request closed without merging**